ngraph/core/include/ngraph/op/experimental_detectron_detection_output.hpp

```diff
@@ -28,9 +28,6 @@ namespace ngraph
     {
         namespace v6
         {
-            /// \brief An operation ExperimentalDetectronDetectionOutput, according to
```

Comment:
Why did you remove doxygen documentation?

Reply:
Because links to openvino/training_extensions repository is outdated. Files mentioned in these descriptions were removed in https://github.com/openvinotoolkit/training_extensions/commit/e7eed680b35acd0bac2060ac1649bea1d76d33c0

Reply:
I propose to add the new description for the class but don't remove full information about the operation.
If we have doxygen description for the operation, it allows to have generate API documentation for the op.

Reply:
Ok, I return and update these descriptions.